### ISSUE TITLE
Lulu 2.9.7 not start after restart

### ISSUE THREAD

**Issue by queermeet-dev** [objective-see/LuLu#672]
Hello!

When I restart my MacBook with MacOS 15.2 then Lulu is not automatically starting and every blocked application can access the Internet.

**Comment by engineeror** [comment]
yup, constant issue. with this app you basically install it once and never update, cause they can't get this right

**Comment by ashhem** [comment]
Having the same issue. Details:

OS: MacOS Sequoia 15.3.1 (24D70)
LuLu Version: 3.1.2
